src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -844,157 +577,38 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
-  const renderShowNametagsToggle = () => (
-    <>
-      <h4 className="italic input-header">
-        Display user names on their avatars
-      </h4>
-      <label className="input-container">
-        <Form.Control as="select" name="showNametags" ref={register} custom>
-          <option value="none">None</option>
-          {/* TODO: Implement Inline state */}
-          {/* <option value="inline">Inline</option> */}
-          <option value="hover">Inline and hover</option>
-        </Form.Control>
-      </label>
-    </>
-  );
-
-  // @debt pass the header into Toggler's 'label' prop instead of being external like this
-  const renderRadioToggle = () => (
-    <div className="toggle-room">
-      <h4 className="italic input-header">Enable venue radio</h4>
-      <Toggler name="showRadio" forwardedRef={register} />
-    </div>
-  );
-
-  const renderRadioStationInput = () => (
-    <div className="input-container">
-      <h4 className="italic input-header">Radio station stream URL:</h4>
-      <input
-        type="text"
-        disabled={disable}
-        name={`radioStations`}
-        ref={register}
-        className="wide-input-block input-centered align-left"
-        placeholder="Radio station URL..."
-      />
-      {errors.radioStations && (
-        <span className="input-error">{errors.radioStations.message}</span>
-      )}
-    </div>
+  const renderHelper = (text: string) => (
+    <p
+      className="small light"
+      style={{ marginBottom: "2rem", fontSize: "16px" }}
```

Comment:
Use classes and avoid inline styles.

Reply:
fixed in def392d3865e2f8d0eaf232f49ef621de2d48e48